_episodes/16-writing-functions.md

```diff
@@ -512,21 +519,21 @@ result of call is: None
 > density. In the model, time takes discrete values 0, 1, 2, ...
 >
 > 1. Define a function called `logistic_map` that takes two inputs: `x`, representing the current
->    population (at time _t_), and a parameter `r=1`. This function should return a value 
->    representing the state of the system (population) at time _t_+1, using the mapping function 
+>    population (at time `t`), and a parameter `r = 1`. This function should return a value 
+>    representing the state of the system (population) at time `t + 1`, using the mapping function:
 >
->    x(t+1) = r * x(t) * [1 - x(t)]
+>    `x[t + 1] = r * x[t] * [1 - x[t]]`
```

Comment:
Hmm, I'm not sold on replacing the round parentheses with square brackets. The round parentheses makes it seem like we are invoking a method, but the square brackets makes it look like we are indexing a dictionary or list, and x isn't either, it's a mathematical function. Perhaps once we have access to [katex](https://github.com/carpentries/styles/pull/573/files) we could render this as a mathematical function instead...

In the meantime, how about replacing `x` here with the generic function `f`?

`f(t+1) = r * f(t) * (1 - f(t))`

Reply:
Sure. Rendering that with Tex would be optimal, but I agree that `f(t)` looks better than `x[t]`.